gtsfm/data_association/data_assoc.py

```diff
@@ -128,6 +128,14 @@ def run(
 
         # dump the 3d point cloud before Bundle Adjustment for offline visualization
         points_3d = [list(triangulated_data.get_track(j).point3()) for j in range(num_accepted_tracks)]
+
+        #create the centered and rotated 3d point cloud - properly aligned with x,y,z axes
+        centered_pc = ellipsoid_utils.center_point_cloud(points)
+        hullSet_points = ellipsoid_utils.compute_convex_hull(centered_pc)
+        polynomial_params = ellipsoid_utils.fit_ls_ellipsoid(hullSet_points)
+        center, axes, rot_matrix = ellipsoid_utils.extract_params_from_poly(polynomial_params)
```

Comment:
It would be useful to have a higher level API that accepts points and provides the aligned_points_3d.